model/org.eclipse.birt.report.model/src/org/eclipse/birt/report/model/elements/interfaces/IInternalReportDesignModel.java

```diff
@@ -345,6 +345,10 @@ public interface IInternalReportDesignModel {
 	 * Word option, list element, add empty line of cell end
 	 */
 	String WORD_LIST_CELL_ADD_EMPTY_PARA = "wordAddEmptyParagraphForListCell"; //$NON-NLS-1$
+	/**
```

Comment:
This is API, please add `@since 4.19 `tag

Reply:
Comment added.